packages/experiments-realm/blog-post.gts

```diff
@@ -607,7 +637,15 @@ export class BlogPost extends CardDef {
               <li class='pub-date'>
                 Published on
                 <time timestamp={{@model.datePublishedIsoTimestamp}}>
-                  {{this.formattedDatePublished}}
+                  {{@model.formattedDatePublished}}
```

Comment:
I've never seen this pattern before and I don't know why I always assumed this to be wrong. 

You use a getter at the root of the CardDef and you use @model.formattedDatePublished to access the model value. This is great bcos you can share these across formats. But, is this different than a computed or not -- do you know? Typically, all my getters exist on the format template.

Reply:
I'm unsure if this is the correct pattern or not. However, it works, and I notice the same pattern in a few cards, such as catalog-entry and leaflet-gtfs. Are you referring to a computed field? This is different from a computed field.

Reply:
I've seen it used before too and it made sense to me. If the getter only applies to a specific format class, then it can be there, but if it applies to all formats, then it can go on the card class I think.